cli/src/pcluster/lib/__init__.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+# with the License. A copy of the License is located at http://aws.amazon.com/apache2.0/
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+# flake8: noqa
```

Comment:
why is the suppression needed? please leave a comment if we can't enable it

Reply:
This is so we can include `ParallelCluster` class even though it isn't used in the `__init__.py` file -- I added a comment explaining that.